test/cluster/test_keyspace_rf.py

```
@@ -92,7 +92,7 @@ def verify_dc_rf(dc: str):
         verify_dc_rf('dc1')
         verify_dc_rf('dc2')
 
-    cql = cluster_con([server.ip_addr], 9042, False,
+    cql = manager.con_gen([server.ip_addr], 9042, False,
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)